systemd-units/plymouth-poweroff.sh.in

```diff
@@ -2,4 +2,5 @@
 
 if [ "$1" = "poweroff" ]; then
 	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
+	sleep 1
```

Comment:
@carlocaione OK. I will merge this, but could you please open another issue in the 2.6 milestone for that?